check_api/src/main/java/com/google/errorprone/ErrorProneAnalyzer.java

```diff
@@ -160,7 +167,17 @@ public void finished(TaskEvent taskEvent) {
     }
   }
 
-  /** Returns true if all declarations inside the given compilation unit have been visited. */
+  /**
+   * Returns true if all declarations inside the given compilation unit have been visited.
+   */
+  private boolean isExcludedPath(URI uri) {
+    Pattern excludedPattern = errorProneOptions.getExcludedPattern();
+    return (excludedPattern != null) && excludedPattern.matcher(uri.toString()).matches();
```

Comment:
Can you use `ASTHelpers.getFileNameFromUri` instead of `URI.toString`? The former has special handling for jar entries, which I think it's possible to see here.